src/components/HelperErrorText/HelperErrorText.tsx

```diff
@@ -66,7 +66,10 @@ export const HelperErrorText = chakra(
         <Box {...props}>
           {isRenderedText ? (
             typeof text === "string" ? (
-              <span dangerouslySetInnerHTML={{ __html: text }} />
+              <Box
+                dangerouslySetInnerHTML={{ __html: text }}
+                __css={styles.innerChild}
+              />
```

Comment:
This element is rendering whether there is helper text or not, so the top margin is still causing spacing issues.  Can line 67 be updated to use this?
```
{(isRenderedText && text) ? (
```

Reply:
Do you have an example of when this happens? I'll add it because the text can be an empty string but it's a required prop and in the storybook pages, i didn't see this being rendered.